com.unity.render-pipelines.high-definition/Runtime/Material/TerrainLit/TerrainLit.shader

```diff
@@ -4,7 +4,7 @@ Shader "HDRP/TerrainLit"
     {
         [HideInInspector] [ToggleUI] _EnableHeightBlend("EnableHeightBlend", Float) = 0.0
         _HeightTransition("Height Transition", Range(0, 1.0)) = 0.0
-        [Enum(Off, 0, From Ambient Occlusion, 1)]  _SpecularOcclusionMode("Specular Occlusion Mode", Int) = 1
+        [HideInInspector] [Enum(Off, 0, From Ambient Occlusion, 1)]  _SpecularOcclusionMode("Specular Occlusion Mode", Int) = 1
```

Comment:
I think we'll have to figure out a way to handle those properties without putting the HideInInspector, because they are not hidden. But for now it doesn't really cause any issue so it's ok